scripts/deploy/deploy-webapp.sh

```diff
@@ -15,6 +15,8 @@ usage() {
     echo "  -d, --deployment-name DEPLOYMENT_NAME  Name of the deployment from a 'deploy-azure.sh' deployment (mandatory)"
     echo "  -a, --application-id APPLICATION_ID    Client application ID (mandatory)"
     echo "  -au, --authority                       Authority to use for client applications that are not configured as multi-tenant. Defaults to (https://login.microsoftonline.com/common) if not specified."
+    echo "  -v  --version VERSION                  Version to set files to (default: 1.0.0)"
```

Comment:
nit:
```suggestion
    echo "  -v  --version VERSION                  Version of deployment (default: 1.0.0)"
```

Reply:
Changed it to match what's in the ps1 file: Version to display in UI